source/configuration_manual/authentication/digest-md5.rst

```diff
@@ -32,12 +32,16 @@ than one realm to choose from. Even if this was NOT one of the choices
 you provided (KMail, others?). In both cases the user never sees the
 advertised realms.
 
-Note that the (badly named) :dovecot_core:ref:`auth_default_realm` setting
+Note that the :dovecot_core:ref:`auth_default_domain` setting
 doesn't work well with Digest-MD5. It doesn't actually specify a default realm
 or have anything to do with Digest-MD5 realms, but rather it appends
-``@auth_default_realm`` to the username if it's missing the ``@domain`` part.
-This will break Digest-MD5 authentication, because the client didn't use the
-``@auth_default_realm`` part in the hash calculations.
+``@auth_default_domain`` to the username if it's missing the ``@domain`` part.
```

Comment:
I meant to remove all of the above text also. Since it's now rather weird about it talking about auth_default_domain not having anything to do with realms, which it of course doesn't anymore after renaming it.